renku/cli/dataset.py

```diff
@@ -624,8 +624,8 @@ def _init(lock, id_queue):
 
         try:
             for p in processing:
-                p.wait()
-                p.get()
+                p.get(timeout=int(os.getenv('RENKU_REQUEST_TIMEOUT', 300)))
```

Comment:
What's the reasoning behind this change? It'll raise a `TimeoutError` if a download takes more than 300s

Reply:
Get internally invokes for the wait, so we don't need both. And I was having some trouble where I tried to import a dataset but entire TTY was just hanging for a very long time, so I quick patched it with this. I can remove the timeout until we get the ticket about it or extend it to 24 hours or something like that.

Reply:
I'll remove it until we find a nice solution.